statchatui/src/Services/api.js

```diff
@@ -0,0 +1,33 @@
+import { GAME_NAMES } from "../Data/GameNames";
+
+const fetchStats = async (formData) => {
```

Comment:
It's a bit unclear reading this what exactly is "formData". Consider either:

a. Adding a comment here so we know what fetchData is. Here's an example: https://stackoverflow.com/posts/6460748/revisions
b. Instead of a single formData object, have the function accept multiple named parameters (to make it more obvious what's passed in).